stetho/src/main/java/com/facebook/stetho/inspector/network/DownloadingAsyncPrettyPrinterFactory.java

```diff
@@ -22,7 +22,7 @@
 public abstract class DownloadingAsyncPrettyPrinterFactory implements AsyncPrettyPrinterFactory {
 
   private final static ExecutorService sExecutorService =
-      AsyncPrettyPrinterExecutorHolder.sExecutorService;
+      AsyncPrettyPrinterExecutorHolder.getExecutorService();
```

Comment:
This doesn't work you are caching the first value of `getExecutorService()`. If you are calling `shutdownNow()` and the re-assigning the value in `AsyncPrettyPrinterExecutorHolder` then you will not have the new executor here.